TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/managers/CoreInstanceManager.java

```diff
@@ -422,6 +422,19 @@ private Proxy startProxy(ProxyGroup group, Base base) {
     public Base getFreeBase(Group group) {
         if (group.isStatic() && group.getBase() == null)
             return null; // A static group has to have a base specified statically
+
+        if (!group.isStatic() && group.getBase() != null){
+            // Start on the specified base if defined
+            Base base = group.getBase();
+
+            // Dont start if not enough memory is available
+            if(base.getAvailableRam() < group.getRam()){
```

Comment:
Use  `if (base.isConnected() && base.isReady())` as well

Reply:
Shouldn't line 441 do exactly what your PR intends to change?
`.filter(base -> group.getBase() == null || group.getBase().equals(base))`